test/ci/docker/dockerfiles/Dockerfile.ngraph_tf.build_ngtf_debian_gcc6_py35

```diff
@@ -0,0 +1,73 @@
+# ==============================================================================
+#  Copyright 2019 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ==============================================================================
+
+# Environment to build and unit-test ngraph-tensorflow-bridge
+
+# Environment to build and unit-test ngraph on debian10 (buster)
```

Comment:
Could you please update the comments to reflect the change to debian:stretch?  Thanks.

Reply:
Oh. Sorry !!